src/tools/jsondocck/src/main.rs

```diff
@@ -54,27 +54,22 @@ pub enum CommandKind {
 }
 
 impl CommandKind {
-    fn validate(&self, args: &[String], command_num: usize, lineno: usize) -> bool {
+    fn validate(&self, args: &[String], lineno: usize) -> bool {
         let count = match self {
-            CommandKind::Has => (1..=3).contains(&args.len()),
-            CommandKind::Count | CommandKind::Is => 3 == args.len(),
-            CommandKind::Set => 4 == args.len(),
+            CommandKind::Has => (..=2).contains(&args.len()),
```

Comment:
This should be `1..=2`, as the old 0-arg form of `@has` (check file exists) is no longer valid

Reply:
:+1: